upt_macports/tests/test_macports_package.py

```diff
@@ -107,15 +107,15 @@ def setUp(self):
     def test_portfile_creation(self, m_open):
         fn = 'upt_macports.upt_macports.MacPortsPackage._render_makefile_template' # noqa
         with mock.patch(fn, return_value='Portfile content'):
-            self.package._create_portfile()
+            self.package._create_portfile('Portfile content')
             m_open.assert_called_once_with('/outdir/Portfile', 'x',
                                            encoding='utf-8')
             m_open().write.assert_called_once_with('Portfile content')
 
     @mock.patch('builtins.open', side_effect=FileExistsError)
     def test_portfile_file_exists(self, m_open):
         with self.assertRaises(SystemExit):
-            self.package._create_portfile()
+            self.package._create_portfile('Portfile content')
```

Comment:
It is nice to update these tests, but maybe we should add a regression test. What do you think?

Reply:
Can you please elaborate what do you mean by regression test....I tried googling but did not understand what would it mean here.

Reply:
Well, there is no test that proves that this bug is actually fixed. We need a test that would fail if this patch was not merged. A test that would simulate a failure of _render_makefile_template and check that no directory has been created.

Reply:
Is this right?